packages/app/src/systems/DApp/pages/TxApprove/TxApprove.tsx

```diff
@@ -1,77 +1,137 @@
-        {!isLoading && !isSent && (
-          <Stack gap="$4">
-            {url && account && <ConnectInfo origin={url} account={account} />}
-            <Stack gap="$2">
-              <HelperIcon as="h2" message="Some message">
-                Assets amount
-              </HelperIcon>
-              {txRequest?.outputs && (
-                <AssetsAmount amounts={getCoinOutputsFromTx(txRequest)} />
+    <>
+      <Layout title="Approve Transaction" isLoading={isLoading}>
+        <Layout.TopBar type={TopBarType.external} />
+        <Layout.Content css={styles.content}>
+          {!isLoading && !approvedTx && (
+            <Stack gap="$4">
+              <Card>
+                <Card.Body
+                  css={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-evenly',
+                  }}
+                >
+                  <Tag css={styles.approveUrlTag} variant="outlined">
+                    <Text as="span" fontSize="sm" color="gray12">
+                      swayswap.io
+                    </Text>
+                  </Tag>
+                  <Text
+                    as="span"
+                    fontSize="sm"
+                    color="gray12"
+                    css={{ fontWeight: '$semibold' }}
+                  >
+                    wants to approve
+                  </Text>
+                </Card.Body>
+              </Card>
+              {account && (
+                <TxFromTo
+                  from={{
+                    type: AddressType.account,
+                    address: account.publicKey,
+                  }}
+                  to={{
+                    type: AddressType.account,
+                    address: outputsToSend[0]?.to.toString(),
+                  }}
+                />
               )}
+              <AssetsAmount
+                amounts={outputsToSend}
+                balanceErrors={groupedErrors?.InsufficientInputAmount}
+                title="Assets to Send"
+              />
+              <TxDetails receipts={receipts} outputAmount={outputAmount} />
             </Stack>
-            {simulateResult && <TxDetails receipts={simulateResult.receipts} />}
-          </Stack>
-        )}
-        {isSent && (
-          <Stack>
-            <Heading as="h4">Transaction sent</Heading>
-            <Text>
-              Transaction sent successfully, you can open your wallet and check
-              its status right now!
-            </Text>
-          </Stack>
-        )}
-      </Layout.Content>
-      <Layout.BottomBar>
-        <Button color="gray" variant="ghost">
-          Close
-        </Button>
-        <Button
-          type="submit"
-          color="accent"
-          onPress={handlers.approve}
-          isLoading={isLoadingTx}
-          isDisabled={isSent}
-        >
-          Send
-        </Button>
-      </Layout.BottomBar>
-    </Layout>
+          )}
+          {approvedTx && (
```

Comment:
This could be a new page instead of a component. We should add a link on this content to open on the block-explorer.

Reply:
it should send to "View Transaction" page, which is not done yet.

can we do this later when #38 gets done and merged?

Reply:
I included the blockExplorer link for now